# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_net-tools.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./slattach.c~	Sat Oct 28 12:59:41 2000
+++ ./slattach.c	Sun May 12 19:19:27 2002
@@ -46,13 +46,7 @@
 #include <getopt.h>
 #include <linux/if_slip.h>
 
-#if defined(__GLIBC__)
-#if __GLIBC__ == 2 && __GLIBC_MINOR__ == 0
-# include <termbits.h>
-#else
-# include <termios.h>
-#endif
-#endif
+#include <termios.h>
 
 #include "pathnames.h"
 #include "net-support.h"
--- ./Makefile~	Sun May 12 19:30:29 2002
+++ ./Makefile	Sun May 12 19:57:29 2002
@@ -188,7 +188,7 @@
 		$(CC) $(LDFLAGS) -o ifconfig ifconfig.o $(NLIB) $(RESLIB)
 		
 nameif:	nameif.o
-		$(CC) $(LDFLAGS) -o nameif nameif.o 
+		$(CC) $(LDFLAGS) -o nameif nameif.o -lcompat
 
 hostname:	hostname.o
 		$(CC) $(LDFLAGS) -o hostname hostname.o $(DNLIB)
